device-registry/src/main/scala/org/genivi/sota/device_registry/Routes.scala

```diff
@@ -122,48 +120,45 @@ class Routes(namespaceExtractor: Directive1[Namespace],
   def api: Route =
```

Comment:
btw all these routes should have their own resource, ie, system info resource and group info resource. 

Also this file should not be named "Routes", that means nothing.

But I guess this can be done in another ticket